### ISSUE TITLE
Link (edit mode) - Bookmarks - not accessible with screen reader

### ISSUE THREAD

**Issue by CarmenHilpert** [ckeditor/ckeditor5#19163]
## 📝 Provide detailed reproduction steps (if any)

1. Open https://ckeditor.com/docs/ckeditor5/latest/features/link.html
2. Insert Link (use button or shorcut)
3. Select "Bookmarks" in the balloon panel

### ✔️ Expected result

The screen reader should announce:

- List
- with 2 entries
- Heritage button

### ❌ Actual result

The Screen reader announces:

- List
- Heritage button

### ❓ Possible solution

- Number of elements = 0: Display only an informational text (already implemented).
- Number of elements = 1: Both `<ul>` and `<li>` have the attribute `role="presentation"` → In screen readers, the button is announced without a list structure.
- Number of elements > 1: `<ul>` and all `<li>` elements have no `role="presentation"` attribute → In screen readers, the list is announced correctly as a list.

## 📃 Other details

* Browser: Edge 138.0.3351.65
* OS: Windows 11 (10.0.22631 Build 22631)
* Screenrader: JAWS 2024
* First affected CKEditor version: v46.0
* Installed CKEditor plugins: see demo page

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.
